setup.py

```diff
@@ -21,4 +21,8 @@
     packages=find_packages(),
     include_package_data=True,
     zip_safe=False,
+    entry_points="""\
+    [console_scripts]
+    devdata = lms.scripts:devdata
+    """,
```

Comment:
This is the standard way of adding command line scripts to a Pyramid app, the same way as builtin scripts like pshell etc work. The `bin/hypothesis` thing we have in h is... :man_shrugging: